### PR TITLE
Fix Add APP parser

### PR DESCRIPTION
**Pull request by jjlopezm**
## Description
This PR solves the problem  of the path discover in parser, so this PR moves the path declaration to a StringLit

Example of use:
ADD APP **'**/my/path/TestAddApp.jar**'** AS testApp WITH com.stratio.addApp.AddAppTest.main
### Testing
- [ ] Unit, integration tests --> XDDddlParserSpec.scala

### Documentation
- [ ] Changelog update
- [ ] Documentation link --> https://stratio.atlassian.net/wiki/display/CROSSDATA/Reference+Guide

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/XDDdlParser.scala [hunk 1]
```diff
@@ -168,11 +170,27 @@ class XDDdlParser(parseQuery: String => LogicalPlan, xDContext: XDContext) exten
         AddJar(xDContext,jarPath.trim)
     }
 
-  protected lazy val addApp: Parser[LogicalPlan] =
-    (ADD ~> APP ~> ident) ~ (AS ~> ident).? ~ (WITH ~> className) ^^ {
-      case jarPath ~ alias ~ cname =>
-        AddApp(xDContext, jarPath.toString, cname, alias)
-    }
+
+//TODO nvestigate how to detect paths with the parser for add app command
```
Comment (miguel0afd) [inline core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/XDDdlParser.scala]: I would say that the use of quotation marks is a final solution, not a temporary one. Therefore, I would remove this TODO

Comment (pfcoperez) [inline core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/XDDdlParser.scala]: @miguel0afd +1

**Comment by pfcoperez** [comment]
:+1:

**Comment by miguel0afd** [comment]
LGTM

**Pull request merged**